### PR TITLE
small fix

### PR DESCRIPTION
**Pull request by kazaamjt**
Update the jenkinsfile.
Related to [IRT #98](https://github.com/inmanta/irt/issues/98)

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -2,7 +2,7 @@ pipeline {
   agent any
   triggers{
     pollSCM '* * * * *'
-    cron("H H(2-5) * * *")
+    cron(BRANCH_NAME == "master" ? "H H(2-5) * * *": "")
```
Comment (arnaudsjs) [inline Jenkinsfile:5]: Could you also remove the `pollSCM` line. It's not required, since we use the Github jenkins plugin.

Comment (kazaamjt) [reply Jenkinsfile:4]: Good point. I also removed some obsolete code and made dev the default pypi index.

File: Jenkinsfile [hunk 2]
```diff
@@ -1,14 +1,13 @@
...
   }
   options { disableConcurrentBuilds() }
-  parameters {
-    booleanParam(name:"pytest_inmanta_dev" ,defaultValue: false, description: 'Changes the index used to install pytest-inmanta to the inmanta dev index')
-  }
   environment {
+    PIP_INDEX_URL='https://artifacts.internal.inmanta.com/inmanta/dev'
+    PIP_PRE="true"
+    PYTEST_INMANTA_DEV="true"
```
Comment (arnaudsjs) [inline Jenkinsfile:10]: The `PYTEST_INMANTA_DEV` environment variable is never used, right?

**Comment by kazaamjt** [comment]
This PR is now obsolete

**Pull request closed without merging**